docs/source/installation/windows.rst

```diff
@@ -0,0 +1,144 @@
+Windows
+=======
+
+The easiest way of installing Manim and its dependencies is by using a
+package manager like `Chocolatey <https://chocolatey.org/>`__ 
+or `Scoop <https://scoop.sh>`__. If you are not afraid of editing
+your System's ``PATH``, a manual installation is not too difficult
+either, though. In fact, if you already have an existing Python
+installation (3.7 or newer), it might be the easiest way to get
+everything up and running.
+
+In case you choose to use one of the package managers, please follow
+their installation instructions
+(`for Chocolatey <https://chocolatey.org/install#install-step2>`__, 
+`for Scoop <https://scoop-docs.now.sh/docs/getting-started/Quick-Start.html>`__)
+to make one of them available on your system.
+
+
+Required Dependencies
+---------------------
+
+Manim requires a recent version of Python (3.7 or above), and ``ffmpeg``
+in order to work.
+
+Chocolatey
+**********
+
+Manim can be installed via Chocolatey simply by running:
+
+.. code-block:: powershell
+
+   choco install manimce
+
+That's it, no further steps required. You can continue with installing
+the :ref:`optional dependencies <win-optional-dependencies>` below.
+
+Scoop
+*****
+
+While there is no recipe for installing Manim with Scoop directly,
+we can install all requirements by running:
+
+.. code-block:: powershell
+
+   scoop install python ffmpeg
+
+and then Manim can be installed by running:
+
+.. code-block:: powershell
+
+   python -m pip install manim
+
+After this, Manim is installed on your system. Continue reading
+the :ref:`optional dependencies <win-optional-dependencies>` section
+below.
+
+Manual Installation
+*******************
+
+As mentioned above, Manim needs a reasonably recent version of
+Python 3 (3.7 or above), and FFmpeg.
+
+**Python:** Head over to https://www.python.org, download an installer
+for Python 3.7 (or newer), and follow its instructions to get Python
+installed on your system.
+
+.. note::
+
+   We have received reports of problems caused by using the version of
+   Python that can be installed from the Windows Store. At this point,
+   we recommend to stay away from the Windows Store and rather install
+   Python directly from https://www.python.org.
+
+**FFmpeg:** In order to install FFmpeg, you can get the source code
```

Comment:
I think we should put the "only if you know what you're doing) after the recommended instructions which should be followed by most users.

Reply:
Done via a5641a2f9a00b072062e7b441c221cd6450fa45d